src/citrine/_serialization/properties.py

```diff
@@ -156,7 +156,11 @@ def __set__(self, obj, value: typing.Union[SerializedType, DeserializedType]):
             value_to_set = self.deserialize(value, base_class=base_class)
 
         if base_class is not None and self.override:
-            getattr(base_class, self.serialization_path).fset(obj, value_to_set)
```

Comment:
This seems to be the hot spot that caused today's dev0. Maybe add some comment here clarifying why the change is being made?

Reply:
Done